src/ralph/data_center/admin.py

```diff
@@ -283,7 +283,7 @@ class DataCenterAssetOperation(OperationViewReadOnlyForExisiting):
 
 
 class DataCenterAssetSecurityInfo(SecurityInfo):
-    url_name = 'datacenter_asset_security_info'
+    url_name = 'security_info'
```

Comment:
yes. if sth will not be in this map then just don't reverse url (and display it as simple text) and log error.